app/javascript/components/enrollment/Enrollment.js

```diff
@@ -184,17 +187,28 @@ class Enrollment extends React.Component {
         this.setState({ index: lastIndex, lastIndex: null });
       });
     } else {
+      let step = 1;
+      if ((this.state.enrollmentState.isolation ? 4 : 5) == index) {
+        step = 2;
+      }
       this.setState({ direction: 'next' }, () => {
-        this.setState({ index: index + 1, lastIndex: null });
+        this.setState({ index: index + step, lastIndex: null });
       });
+      if (index + step == maxEnrollmentSteps) {
+        this.setState({ reviewing: true });
+      }
     }
   };
 
   previous = () => {
     window.scroll(0, 0);
     let index = this.state.index;
+    let step = 1;
```

Comment:
can we clean this logic up to one line?

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/327ffa10a1eb5121c33a8345b00bf09b7741ca4a